### DISCUSSION TITLE
update to new version lose database

### DISCUSSION THREAD

**Discussion by xiangxud** [taosdata/TDengine#7957] [Q&A]
When I update to the latest version (GIT clone from default dev branch), my database loses one of the multiple databases I created in the old version。

Why? How do I restore this database。
Is the version number unchanged？
 taos -C
taos global config:
==================================
 serverPort:             6030 
 arbitrator:              
 maxNumOfDistinctRes:    10000000 
 compressMsgSize:        -1 
 compressColData:        -1 
 maxSQLLength:           1048576(byte)
 maxWildCardsLength:     100(byte)
 maxRegexStringLen:      128(byte)
 maxNumOfOrderedRes:     100000 
 keepColumnName:         0 
 timezone:                
 locale:                  
 charset:                 
 gitinfo:                dfc46ebc1a465ac5d2f9ea549edd4a89629843ec 
 gitinfoOfInternal:      dfc46ebc1a465ac5d2f9ea549edd4a89629843ec 
 buildinfo:              Built at 2021-09-16 12:39:43 
 version:                2.1.7.2

**Comment by aloscha1** [comment]
It appears that there is a problem with the update process that is causing the loss of one of your databases. To restore the missing database, you can try to check if the backup files are available and restore them. Additionally, you may want to review the upgrade process to ensure that all databases are migrated properly.

The version number you provided is 2.1.7.2 and appears to be unchanged. However, it is always a good idea to double check the version after the update to make sure it is the same or if there are any changes.

**Comment by yu285** [comment]
The official support for version 2.x has been discontinued. We recommend that you use version 3.x to ensure a good user experience.
